baseapp/baseapp.go

```diff
@@ -329,20 +323,27 @@ func (app *BaseApp) runTx(isCheckTx bool, txBytes []byte, tx sdk.Tx) (result sdk
 		ctx = app.ctxDeliver.WithTxBytes(txBytes)
 	}
 
-	// TODO: override default ante handler w/ custom ante handler.
-
 	// Run the ante handler.
 	newCtx, result, abort := app.anteHandler(ctx, tx)
-	if isCheckTx || abort {
+	if abort {
```

Comment:
This is correct, but did you make sure all implementations handle checktx properly?

Reply:
What implementations are you referring to?

In the worst case all implementations will now just do extra work against the CheckTx state. So in the case of the EVM, it would now run the full transaction through the EVM instead of returning beforehand.

Reply:
CheckTx is supposed to return a non-zero count.
I refer to at least bank/handler. But also other Handlers that are in the codebase.